src/java.base/share/classes/jdk/internal/classfile/components/package-info.java

```diff
@@ -68,30 +79,36 @@
  * {@snippet lang="java" class="PackageSnippets" region="allPackageRemap"}
  *
  * <h3>{@link CodeLocalsShifter}</h3>
- * {@link CodeLocalsShifter} is a {@link jdk.internal.classfile.CodeTransform} shifting locals to
- * newly allocated positions to avoid conflicts during code injection.
- * Locals pointing to the receiver or to method arguments slots are never shifted.
- * All locals pointing beyond the method arguments are re-indexed in order of appearance.
+ * {@link CodeLocalsShifter} is a {@link jdk.internal.classfile.CodeTransform}
+ * shifting locals to newly allocated positions to avoid conflicts during code
+ * injection. Locals pointing to the receiver or to method arguments slots are
+ * never shifted. All locals pointing beyond the method arguments are re-indexed
+ * in order of appearance.
  * <p>
  * Sample of code transformation shifting all locals in all methods:
  * {@snippet lang="java" class="PackageSnippets" region="codeLocalsShifting"}
  *
  * <h3>{@link CodeRelabeler}</h3>
- * {@link CodeRelabeler} is a {@link jdk.internal.classfile.CodeTransform} replacing all occurences
- * of {@link jdk.internal.classfile.Label} in the transformed code with new instances.
- * All {@link jdk.internal.classfile.instruction.LabelTarget} instructions are adjusted accordingly.
+ * {@link CodeRelabeler} is a {@link jdk.internal.classfile.CodeTransform}
+ * replacing all occurences of {@link jdk.internal.classfile.Label} in the
+ * transformed code with new instances.
+ * All {@link jdk.internal.classfile.instruction.LabelTarget} instructions are
+ * adjusted accordingly.
  * Relabeled code graph is identical to the original.
  * <p>
- * Primary purpose of {@link CodeRelabeler} is for repeated injections of the same code blocks.
- * Repeated injection of the same code block must be relabeled, so each instance of
- * {@link jdk.internal.classfile.Label} is bound in the target bytecode exactly once.
+ * Primary purpose of {@link CodeRelabeler} is for repeated injections of the
+ * same code blocks.
+ * Repeated injection of the same code block must be relabeled, so each instance
+ * of {@link jdk.internal.classfile.Label} is bound in the target bytecode
+ * exactly once.
  * <p>
  * Sample transformation relabeling all methods:
  * {@snippet lang="java" class="PackageSnippets" region="codeRelabeling"}
  *
  * <h3>Class Instrumentation Sample</h3>
- * Following snippet is sample composition of {@link ClassRemapper}, {@link CodeLocalsShifter}
- * and {@link CodeRelabeler} into fully functional class instrumenting transformation:
+ * Following snippet is sample composition of {@link ClassRemapper}, {@link
+ * CodeLocalsShifter} and {@link CodeRelabeler} into fully functional class
+ * instrumenting transformation:
  * {@snippet lang="java" class="PackageSnippets" region="classInstrumentation"}
  */
 package jdk.internal.classfile.components;
```

Comment:
watch out for newline

Reply:
I'm not quite sure I understand what is wrong with this javadoc fragment.
Thanks.